tests/foreman/ui/test_bookmarks.py

```diff
@@ -192,16 +268,30 @@ def test_negative_create_bookmark_same_name(self):
 
         @Assert: Error notification - name already taken, Bookmark is not
         created (not listed)
-
-        @Status: Manual
         """
-        for entity in BOOKMARK_ENTITIES:
-            if entity.get('setup'):
-                create_entity(entity['setup'])
-            pass
+        for entity in self.entities:
+            with self.subTest(entity):
+                with Session(self.browser) as session:
+                    name = gen_random_string()
+                    session.nav.go_to_select_org(self.org_.name)
+                    ui_lib = getattr(self, entity['name'].lower())
+                    ui_lib.create_a_bookmark(
```

Comment:
ohh, it is a negative scenario. Do we have any errors after second call, if not - then collapse two calls into one, if yes - please check that message